src/lib/__tests__/sortMenu-test.js

```diff
@@ -1,14 +1,12 @@
 import sortMenu from '../sortMenu';
 
-function stringify(array) {
-  let string = '';
-  array.forEach(obj => {
-    string += JSON.stringify(obj);
-  });
-  return string;
-}
-
 describe('sortMenu', () => {
+  function check(actual, expected) {
```

Comment:
how does this differ from `expect(actual).toEqual(expected)` ?

Reply:
it's 2 different arrays